ipatests/test_webui/ui_driver.py

```diff
@@ -823,6 +823,41 @@ def del_multivalued(self, name, value, parent=None):
 
         assert clicked, 'Value was not removed: %s' % value
 
+    def undo_multivalued(self, name, value, parent=None):
+        """
+        Undo value in multivalued textbox.
+        """
+        if not parent:
```

Comment:
@stlaz This is really good question. I hold on the code style which is introduced in this file. Of course, I agree with you it could be written better, more pythonic way.
I will addressed yoour comment but... I will change all occurrences in this file, do you agree?